app/client/src/components/Message.vue

```diff
@@ -92,6 +92,7 @@
 
     .message {
         display: flex;
+	flex: 0 0 auto;
```

Comment:
The added line uses a single tab instead of 4*2 spaces (as used for the rest of the file), this can be visualized by going there: https://github.com/skychatorg/skychat/blob/cfd8d30bd44dd6759551cbf1c6b1935b49c3659f/app/client/src/components/Message.vue?ts=1#L95

Could you please update this line to be aligned and use spaces instead, so I can approve the PR?

Reply:
Hello, silly me.
Done. 👍